WordPress/Classes/ViewRelated/Site Creation/Verticals/VerticalsWizardContent.swift

```diff
@@ -228,5 +204,14 @@ extension VerticalsWizardContent: UITableViewDataSource, UITableViewDelegate {
         return vertical.isNew ? NewVerticalCell.cellReuseIdentifier() : VerticalsCell.cellReuseIdentifier()
     }
 
+    private func addBorder(cell: UITableViewCell, at: IndexPath) {
```

Comment:
We don't have that many rows, but perhaps these borders be cleared in `prepareForReuse()`?